homeassistant/components/flo/entity.py

```diff
@@ -0,0 +1,69 @@
+"""Base entity class for Flo entities."""
+
+from typing import Any, Dict
+
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN as FLO_DOMAIN, SIGNAL_DEVICE_UPDATED
+from .device import FloDevice
+
+
+class FloEntity(Entity):
+    """A base class for Flo entities."""
+
+    def __init__(self, unique_id: str, name: str, device: FloDevice, **kwargs):
+        """Init Flo entity."""
+        self._unique_id: str = unique_id
+        self._name: str = name
+        self._device: FloDevice = device
+        self._state: Any = None
+
+    @property
+    def name(self) -> str:
+        """Return Entity's default name."""
+        return self._name
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return self._unique_id
+
+    @property
+    def device_info(self) -> Dict[str, Any]:
+        """Return a device description for device registry."""
+        return {
+            "identifiers": {(FLO_DOMAIN, self._device.id)},
```

Comment:
Can you add `connections` with the mac address? That way device trackers can link up with it.

Reply:
sure can

Reply:
done